eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -62,6 +62,8 @@ parameters:
       Preview:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
         ServiceConnection: azure-sdk-tests
+        # Temporary fix until an eng/common config for Preview can be merged
+        SubscriptionConfigurationFilePaths: []
```

Comment:
What's requiring this change? I don't understand the comment. What is a "config for Preview"?

Reply:
that was introduced by Daniel in my PR based on the comment this is needed for the Preveiw cloud , whatever thast is